Orange/canvas/report/owreport.py

```diff
@@ -191,6 +191,10 @@ def _remove_item(self, row):
         self.table_model.removeRow(row)
         self._build_html()
 
+    def _clear(self):
```

Comment:
Oops, in my code it's `clear`, without underscore. I don't know how this happened; I now (force-)pushed again.